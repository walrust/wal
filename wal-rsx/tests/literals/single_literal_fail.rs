use wal_rsx::rsx;

fn main() {
    rsx! { b"val" };
}
