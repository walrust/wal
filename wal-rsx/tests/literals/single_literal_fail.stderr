error: Byte string literals are not supported
 --> tests/literals/single_literal_fail.rs:4:12
  |
4 |     rsx! { b"val" };
  |            ^^^^^^
